code/modules/admin/verbs/freeze.dm

```diff
@@ -38,18 +38,18 @@ GLOBAL_LIST_EMPTY(frozen_atom_list) // A list of admin-frozen atoms.
 		GLOB.frozen_atom_list += src
 
 		var/obj/effect/overlay/adminoverlay/AO = new
-		if(skip_overlays)
+		if(!skip_overlays)
 			overlays += AO
 
 		anchored = TRUE
 		admin_prev_sleeping = AmountSleeping()
-		frozen = AO
+		frozen = TRUE
```

Comment:
if you are converting this to a bool, remove all the instances where its set to `null`

Reply:
Oops, I meant to revert that when I was doing some testing with why humans were so funky with their freezing